services/configServices/__tests__/NetlifyTomlService.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios")
+const { Base64 } = require("js-base64")
+
+const validateStatus = require("@utils/axios-utils")
+
+const {
+  netlifyTomlContent,
+  netlifyTomlHeaderValues,
+} = require("@root/fixtures/netlifyToml")
+
+const { GITHUB_BUILD_ORG_NAME, GITHUB_BUILD_REPO_NAME } = process.env
+
+jest.mock("axios")
+
+describe("Auth Service", () => {
+  const accessToken = "test-token"
+
+  const reqDetails = { accessToken }
+
+  const {
+    NetlifyTomlService,
+  } = require("@services/configServices/NetlifyTomlService")
+
+  const service = new NetlifyTomlService()
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe("read", () => {
+    it("Reading the netlify.toml file works correctly", async () => {
```

Comment:
nit: it `should read successfully when the netlify.toml is valid` (cos i alw read the test description with the "it" ._.)

Reply:
Makes sense, [a6cdec7](https://github.com/isomerpages/isomercms-backend/pull/357/commits/a6cdec7b64e02ef1769b77c35a2cfcfa3b882986)